spec/services/nytimes_service_spec.rb

```diff
@@ -9,16 +9,14 @@
 			end
 
 			expect(Article.first.title).to(
-				eq("California Imposes Water Restrictions, in First for State")
+				eq("California Drought Tests History of Endless Growth")
 			)
 			expect(Article.first.url).to(
-				eq("http://www.nytimes.com/2015/04/02/us/california-imposes-first-" +
-					 "ever-water-restrictions-to-deal-with-drought.html")
+				eq("http://www.nytimes.com/2015/04/05/us/california-drought-tests-history" + 
```

Comment:
Line is too long. [81/80]<br>Indent the first parameter one step more than the previous line.<br>Tab detected.<br>Trailing whitespace detected.